databricks/koalas/utils.py

```diff
@@ -75,7 +75,7 @@ def combine_frames(this, *args, how="full"):
         # If the same named index is found, that's used.
         for this_column, this_name in this_index_map:
             for that_col, that_name in that_index_map:
```

Comment:
can you rename `that_col` to `that_column`?

Reply:
thanks. i fixed it